src/components/Map/Controls/index.tsx

```diff
@@ -16,31 +16,25 @@ import {
   Box,
 } from "@mui/material";
 import { LocaleSwitchComponent } from "../../LocaleSwitch/LocaleSwitch";
-// import SearchIcon from "@mui/icons-material/Search";
-// import WifiTetheringErrorIcon from "@mui/icons-material/WifiTetheringError";
 import Diversity1Icon from "@mui/icons-material/Diversity1";
 import { FilterButtonComponent } from "../../Button/Filter";
 import AddIcon from "@mui/icons-material/Add";
 import RemoveIcon from "@mui/icons-material/Remove";
 import LayersIcon from "@mui/icons-material/Layers";
+import SearchIcon from "@mui/icons-material/Search";
+import WifiTetheringErrorIcon from "@mui/icons-material/WifiTetheringError";
 import { useMap } from "react-leaflet";
 import { Control } from "./Control";
 import { LayerButton } from "./LayerButton";
-import {
-  FilterComponent,
-  IFilterElement,
-  createUseFilter,
-} from "../../Filter/Filter";
 import { useHelpView } from "../../UserGuide/UserGuide";
-import { useCallback, useEffect, useState } from "react";
-import { useRouter } from "next/router";
-import { usePrevious } from "@/hooks/usePrevious";
 import { useTranslation } from "next-i18next";
 import DoubleClickStopPropagation from "@/components/DoubleClickStopPropagation";
-
-export const usePoiFilter = createUseFilter();
-// const useHelpFilter = createUseFilter(tempFilterData2);
-// const useSearchFilter = createUseFilter(tempFilterData3);
+import { FilterDisasterVictim } from "@/features/location-filters/components/FilterDisasterVictim";
+import { FilterHelpRequest } from "@/features/location-filters/components/FilterHelpRequest";
+import { FilterService } from "@/features/location-filters/components/FilterService";
+import { useDisasterVictimFilter } from "@/features/location-filters/stores/useDisasterVictimFilter";
+import { useHelpRequestFilter } from "@/features/location-filters/stores/useHelpRequestFilter";
+import { useServiceFilter } from "@/features/location-filters/stores/useServiceFilter";
```

Comment:
import {  FilterDisasterVictim, FilterHelpRequest, FilterService, ...} from "features/location-filters" 

what do you think?